sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSessionManagerTest.java

```diff
@@ -154,34 +116,6 @@ void afterEach(TestInfo testInfo) throws Exception {
         if (sessionManager != null) {
             sessionManager.close();
         }
-
-        if (connectionProcessor != null) {
-            connectionProcessor.dispose();
-        }
-    }
-
-    @Test
-    void properties() {
-        // Arrange
-        ReceiverOptions receiverOptions = createUnnamedSessionOptions(ServiceBusReceiveMode.PEEK_LOCK, 1,
-            MAX_LOCK_RENEWAL, false, 5, SESSION_IDLE_TIMEOUT);
-        sessionManager = new ServiceBusSessionManager(ENTITY_PATH, ENTITY_TYPE, connectionCacheWrapper,
-            messageSerializer, receiverOptions, CLIENT_IDENTIFIER, NOOP_TRACER);
-
-        // Act & Assert
-        assertEquals(CLIENT_IDENTIFIER, sessionManager.getIdentifier());
-    }
-
-    @Test
-    void receiveNull() {
-        // Arrange
-        ReceiverOptions receiverOptions = createUnnamedSessionOptions(ServiceBusReceiveMode.PEEK_LOCK, 1,
-            MAX_LOCK_RENEWAL, false, 5, SESSION_IDLE_TIMEOUT);
-        sessionManager = new ServiceBusSessionManager(ENTITY_PATH, ENTITY_TYPE, connectionCacheWrapper,
-            messageSerializer, receiverOptions, CLIENT_IDENTIFIER, NOOP_TRACER);
-
-        // Act & Assert
-        StepVerifier.create(sessionManager.receive()).expectError(NullPointerException.class).verify(DEFAULT_TIMEOUT);
```

Comment:
In v2 `ServiceBusSingleSessionManager`, the `receive()` is guaranteed to return a Flux never null.